src/main/java/dto/UserGenerator.java

```diff
@@ -0,0 +1,12 @@
+package dto;
+
+public class UserGenerator {
+    public static User getNewUser() {
+        return User
+            .builder()
+            .name("Robert")
```

Comment:
Please move this hardcoded values to property file

Reply:
fixed